src/solvers/lowering/byte_operators.cpp

```diff
@@ -111,7 +111,7 @@ static array_exprt unpack_array_vector(
     // recursively unpack each element until so that we eventually just have an
     // array of bytes left
     exprt sub =
-      unpack_rec(element, little_endian, nil_exprt(), max_bytes, ns, true);
+      unpack_rec(element, little_endian, nullopt, max_bytes, ns, true);
```

Comment:
Usually `{}` rather than `nullopt`

Reply:
We seem to have a small number of uses, but agreed that `{}` is more common and thus to be used.